Cost-Center-Creation-Neo/application/src/main/java/com/sap/opensap/CostCenterCreationServlet.java

```diff
@@ -0,0 +1,150 @@
+package com.sap.opensap;
+
+import org.joda.time.LocalDate;
+import org.slf4j.Logger;
+
+import javax.annotation.Nullable;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+import com.sap.cloud.sdk.cloudplatform.logging.CloudLoggerFactory;
+
+import com.sap.cloud.sdk.s4hana.connectivity.ErpEndpoint;
+import com.sap.cloud.sdk.s4hana.connectivity.exception.QueryExecutionException;
+import com.sap.cloud.sdk.s4hana.connectivity.rfc.BapiQuery;
+import com.sap.cloud.sdk.s4hana.connectivity.rfc.BapiQueryResult;
+import com.sap.cloud.sdk.s4hana.connectivity.rfc.Table;
+
+@WebServlet("/cost-center-creation")
+public class CostCenterCreationServlet extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    private static final Logger logger = CloudLoggerFactory.getLogger(CostCenterCreationServlet.class);
+
+    @Override
+    protected void doGet(final HttpServletRequest request, final HttpServletResponse response)
```

Comment:
Shouldn't that be done on doPost? Creation of cost centers on HTTP get is confusing.